src/middleware/boilerplates/runner/createServices.js

```diff
@@ -66,10 +67,12 @@ function createServices(broker) {
         origin: '*',
         exposedHeaders: '*'
       },
-      routes: [...LdpRoutes, ...SparqlEndpointRoutes, ...WebIdRoutes, ...ActivityPubRoutes],
-      defaultLdpAccept: 'text/turtle'
+      routes: [...SparqlEndpointRoutes, ...WebIdRoutes, ...ActivityPubRoutes, ...WebhooksRoutes]
```

Comment:
ReferenceError: WebhooksRoutes is not defined

Reply:
:+1: